uws/UWS/models.py

```diff
@@ -10,6 +10,25 @@ def _parse_bool(self, value):
             return False
         return value
 
+    def _find_ns_uws(self, namespaces):
```

Comment:
please change all occurrences of `ns` to `namespace`. Being more verbose helps readability.

And the function should be called `_find_uws_namespace` since this is the order you used everywhere below.